.gitignore

```diff
@@ -14,6 +14,7 @@ node_modules/
 ajax/libs/*/*/.bot_cant_auto_add
 
 # folders need to be ingored due to auto auto-update mechanism
+ajax/libs/jquery-price-format/2.0.0/
```

Comment:
What is this for? There is no `2.0.0` tag in repo and you added v2.0, so no need to ignore it,